openshift/Makefile

```diff
@@ -21,6 +21,7 @@ export GO_BUILD_LDFLAGS := -s -w -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)'
 verify: ## Run downstream-specific verify
 	$(MAKE) tidy fmt generate -C $(DIR)/../
 	$(MAKE) manifests
+	$(MAKE) -C tests-extension verify
```

Comment:
Could we remove from here 
We have a sanity for OTE

Reply:
I'm not seeing any other CI that invokes this.

Reply:
Yes, it is: https://github.com/openshift/operator-framework-operator-controller/pull/441#issuecomment-3192212513